src/app/pages/news/9.3/06-public-api.md

```diff
@@ -1,10 +1,20 @@
 ## Public API {#publicApi93}
 
-- powerful APIs: serving 10 years of project experience
+With twenty years of project experience we knew what customers expect from a powerful process automation platform.
+The newly added API makes process development even easier. 
+
```

Comment:
I'd link the most prominent entry points which are mentioned in the text directly to the concret Public API page.
-> so that customers can directly click on 'ISecurity' and browse it's API